@here/harp-utils/lib/MathUtils.ts

```diff
@@ -163,4 +165,44 @@ export namespace MathUtils {
             time < 0.5 ? 4 * time * time * time : (time - 1) * (2 * time - 2) * (2 * time - 2) + 1;
         return startValue + (endValue - startValue) * timeValue;
     }
+
+    /**
+     * Round number to a specified number of fractional digits, using 'round to nearest' convention.
+     *
+     * Midpoint values of less significant fraction digit (digit on position 'digits + 1' after
+     * comma) are promoting - increase previous digit - if they are greater or equal to 5.
+     *
+     * @note Function accepts negative values using correct arithmetic rounding, such as the correct
+     * numbers for `roundFraction(-0.5, 0)` will be `-1` and for `roundFraction(-1.55, 1)` will
+     * be `-1.6`.
+     * @note If you accidentally pass floating point number (with fraction) to @param digits
+     * this value will be _truncated_, but negative number of digits is not allowed.
+     *
+     * @param value The floating point value to be rounded.
+     * @param digits The number of fractional digits to leave (accuracy after comma).
+     */
+    export function roundFraction(value: number, digitsNum: number = 10) {
```

Comment:
I think this code is a bit over engineered for what we want to do (i.e. `Math.round(zoomLevel * 10e10) / 10e10)`).
Do we gain anything compared to the old simple approach?

Reply:
Just a precision and full control over number of digits + support to negative values.

Reply:
Please keep things simple!

We should not implement sth. b/c maybe somebody might need this in the future.
I cannot imagine a scenario where somebody would need rounding of negative numbers.